api/helpers/game-states/create-socket-event.js

```diff
@@ -14,19 +15,29 @@ module.exports = {
     gameState: {
       type: 'ref',
       description: 'Game state object',
-      required: true,
     },
   },
 
   fn: async function ({ game, gameState }, exits) {
     try {
+      if (!game.status === GameStatus.CREATED && !gameState) {
```

Comment:
added this check here so we can remove the required attribute and still use this helper if the user is not yet in a game

Reply:
This feels strange. Why would we call this method if there is no gamestate? And why restrict it to only games that are started but not finished ones?

Reply:
This is so we can use this method for games not started yet in the lobby, this check basically makes sure that there is a gamestate available for every other time this will be called besides when we are in a lobby

Reply:
It deff should be !== though 
```suggestion
      if (game.status !== GameStatus.CREATED && !gameState) {
```